x/ccv/no_valupdates_genutil/doc.go

```diff
@@ -0,0 +1,8 @@
+/*
+Package staking defines a "wrapper" module around the Cosmos SDK's native
+x/staking module. In other words, it provides the exact same functionality as
```

Comment:
```suggestion
x/genutil module. In other words, it provides the exact same functionality as
```
?

Also, slightly later in this comment, this module does not override `EndBlock` (?)

Reply:
Thanks, good catch